js/defaults_dialog.js

```diff
@@ -255,36 +335,186 @@ helper.defaultsDialog = (function() {
                 },
                 {
                     key: "failsafe_mission",
-                    value: "OFF"
+                    value: "ON"
                 },
                 {
                     key: "nav_wp_radius",
-                    value: 3000
+                    value: 1500
```

Comment:
This is quite a small radius for a fixed wing. If there are strong winds then the WP can easily be missed leading to a sort of loiter around the WP.

Reply:
default is 1m in the past. Pawel raised it to 30m but IMHO that was too much. I usually set my planes all to 10m. 15m should be a good compromise. Kepp in mind that we don't use compass on planes and inav compensates or wind. A radius of 15m is still a 30m wide target to hit the waypoint. Never had any failed waypoint mission even with 10m before.

Reply:
oh and 30m can be too big for aerial mapping missions when you want to fully cover ground. it makes a big difference if the pathes can be offset by 60m.

Reply:
My bad, I misinterpreted this setting as 1.5 meter. I think 10-15 meters would be fine.